### ISSUE TITLE
find and replace panel stops working properly after editing one of the results

### ISSUE THREAD

**Issue by booc0mtaco** [atom/find-and-replace#960]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

If you edit a line after doing find/replace search, you can no longer click find-and-replace searches in the panel to navigate to other search results. You also get a console error in Atom.app .

### Steps to Reproduce

1. Open a file and engage full find/replace mode ( + shift + f)
2. search for a result that appears in multiple files
3. in the search result panel, click on one of the results, this will take you to it in the appropriate file
4. comment out that line using "Edit => Toggle Comments"
5. click on another search result further down in the same file

**Expected behavior:**

It will go to the appropriate line and highlight the result

**Actual behavior:** 

it highlights the previously commented-out line instead. It "does nothing". Once it gets into this behavior, a restart is required to restore normal find behavior.

Opening the atom console also shows this error:

```
Uncaught (in promise) TypeError: Cannot read property '0' of undefined
    at atom.workspace.open.then.editor (/Applications/Atom.app/Contents/Resources/app/node_modules/find-and-replace/lib/project/results-view.js:473:58)
```

**Reproduces how often:**

Happens every time.

### Versions

You can get this information from copy and pasting the output of `atom --version` and `apm --version` from the command line. Also, please include the OS and what version of the OS you're running.

#### Atom
Atom    : 1.20.1
Electron: 1.6.9
Chrome  : 56.0.2924.87
Node    : 7.4.0

#### OS

macOS Sierra 10.12.6 (16G29)

#### Core Plugin

find-and-replace : 0.209.5

### Additional Information

Any additional information, configuration or data that might be necessary to reproduce the issue.

**Comment by Ben3eeE** [comment]
Thanks for taking the time to contribute!

We noticed that this is a duplicate of https://github.com/atom/find-and-replace/issues/953. This issue has been fixed on Atom 1.21 beta.

Because we treat our issues list as [the Atom team's backlog](https://en.wikipedia.org/wiki/Scrum_(software_development)#Product_backlog), we close duplicates to focus our work and not have to touch the same chunk of code for the same reason multiple times. This is also why we may mark something as duplicate that isn't an exact duplicate but is closely related.

For information on how to use GitHub's search feature to find out if something is a duplicate before filing, see [the How Can I Contribute? section of the Atom CONTRIBUTING guide](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-can-i-contribute).
